EdFi.Buzz.Installer/eng/README.md

```diff
@@ -0,0 +1,105 @@
+- **API:** GraphQL-based API for supporting the Ed-Fi Buzz solution.
+- **UI:** Project Buzz Web Application..
+
+## Prerequisites
+We recommend that the following prerequisites are installed on the machine that you are going to run the scripts on.
+
+1. Node Js LTS version https://nodejs.org/en/
+2. Postgresql
+
+## Setup Instructions
+1. Edit the configuration file to include the values according to your installation environment.
+2. Run as administrator the installation script 
+
+
+
+### Configuration file
+
+The configuration variables for the installation are in the **'Windows\configuration.json'** file. In this file you can update the values to adapt them before the installation.
+
+The variables that can be configured are detailed below.
+
+#### General configuration
+- **idProvider:** Authentication provider. Valid values: google, adfs.
+- **googleClientId:** If the authentication provider is google, you must set in this field the corresponding googleClientId when you created the Web Application in Google Developers Console.
+- **adfsClientId:** If the authentication is validated using adfs, you must inclute the clientId.
+- **adfsTenantId:** If the authentication is validated using adfs, you must inclute the adfsTenantId.
+
+#### postgresDatabase
+
+You need to update the Postgres database connection string according to your configuration. You can configure the server, credentials or name to connect to the postgres database.
+- **host:** Database server.
+- **port:** Postgres port (E.g. 5432)
+- **username:** User to connect to the database
+- **password:** Password to connect to the database
+- **database:** Database name. (e.g. "edfi_buzz")
+
+#### sqlServerDatabase
+
+You can configure the server, credentials or name to connect to the Sql Server ODS database.
+- **host:** Sql Server name or IP.
+- **port:** SQL database port.
+- **username:** SQL Server database user name.
+- **password:** SQL Server database.
+- **database:** ODS database. AMT must be installed in this database.
+- **UseIntegratedSecurity:** If you use integrated security. Allowed values: true, false.
+
+#### etl
+
+Options to configure the ETL to load from the database or file to the postgres database.
+- **datasourceFormat:** If the data source corresponds to the Analytics Middle Tier (amt) or direct views of the ODS tables. Allowed values: amt, ods.
+- **odsDataStandard:** Database standard to use, if it is Data Standard 2 or 3.x. Allowed values: ds2, ds3.
+
+#### api
+- **url:** API Url.
+- **port:** API port.
+
+#### ui
+- **port:** Port to access UI.
+
+### Instalation
+Instalation script:
+- It requires administrator privileges.
+- It receives the configPath Full path to a JSON document containing configuration settings for Buzz. Defaults to **.\configuration.json** in the same directory.
+
+The installation script must be run in PowerShell as an administrator.
```

Comment:
I would add that the script will not run on PowerShell Core because WebAdministration does not load properly.